build/__nodeVersions.sh

```diff
@@ -1,15 +1,15 @@
 # This file was auto-generated from 'constants.yaml'. Changes may be overridden.
 
-NODE_RUNTIME_BASE_TAG='20220128.5'
-YARN_VERSION='1.22.15'
-YARN_MINOR_VERSION='1.17'
+NODE_RUNTIME_BASE_TAG='20220128.5' #TODO: Depends on my PR creation date?
```

Comment:
@samruddhikhandale overall looking good! You'll nee to run corresponding platform base image pipeline for upgrading the runtime version: https://devdiv.visualstudio.com/DevDiv/_build?pipelineNameFilter=oryx

For example: Run pipeline `Oryx-BaseImages-Node` and target your branch. If this run successfully, you'll need change all corresponding base image tags in the repo. 

Here is documentation on how to release new platforms: https://microsoft.sharepoint.com/:o:/r/teams/ProjectOryx/_layouts/15/Doc.aspx?sourcedoc=%7Bbbadf169-c181-4007-8661-fc5d27398240%7D&action=edit&wd=target(Welcome.one%7C4dea46ae-e936-4365-894a-fca0f242e3e7%2FOryx%20%E2%80%93%20How%20to%27s%7Cf52fa59f-9626-4841-9c24-e3e13de1d31b%2F)&share=IgFp8a27gcEHQIZh_F0nOYJAAQnhR7emtbuTBe-p9_UIV10&cid=7ed2fb38-5588-43b1-82bc-f54d976bbe35